build kanto suite-connector and service configuration, add README

frankplus opened the pull request:
- add Kanto suite-connector as git submodule
- add build scripts and build metadata to build Kanto suite-connector in the OpenHarmony build system
- add README
- add service to start kanto at system startup

Commit:
Add build script and build metadata for Kanto suite-connector

Signed-off-by: Francesco Pham <francesco.pham@huawei.com>

Commit:
add kanto suite-connector git submodule

Signed-off-by: Francesco Pham <francesco.pham@huawei.com>

Commit:
Update README

Signed-off-by: Francesco Pham <francesco.pham@huawei.com>

Commit:
BUILD.gn: install artifacts into image

Signed-off-by: Francesco Pham <francesco.pham@huawei.com>

Commit:
add config.json and init cfg to start suite-connector at boot

Signed-off-by: Francesco Pham <francesco.pham@huawei.com>

Commit:
properly set up log directory

Signed-off-by: Francesco Pham <francesco.pham@huawei.com>

Commit:
add etc/hosts file because of error resolving localhost

Signed-off-by: Francesco Pham <francesco.pham@huawei.com>

Commit:
update README

Signed-off-by: Francesco Pham <francesco.pham@huawei.com>

Commit:
set kanto service user to kanto 

Signed-off-by: Francesco Pham <francesco.pham@huawei.com>

Stefan-Schmidt commented on README.md:
````diff
@@ -1 +1,59 @@
+- Secondly, clone this repository into `//third_party/kanto`
+
+```console
+$ git clone --recurse-submodules  https://github.com/frankplus/third_party_kanto.git ./third_party/kanto
````
This should change to the new URL

frankplus replied:
fixed

Stefan-Schmidt commented on README.md:
````diff
@@ -1,59 +1,106 @@
 
 ```console
 $ git clone --recurse-submodules  https://github.com/frankplus/third_party_kanto.git ./third_party/kanto
+$ git clone --recurse-submodules  https://github.com/frankplus/third_party_mosquitto.git ./third_party/mosquitto
````
Same for this URL

frankplus replied:
fixed

Stefan-Schmidt commented on README.md:
```diff
@@ -1,59 +1,106 @@
+   "enable_ramdisk": true,
+   "enable_absystem": false,
+-  "build_selinux": true,
++  "build_selinux": false,
```
Do we still need to disable this with the  "secon" : "u:r:kanto:s0" change in the next commit?

frankplus replied:
updated instructions

Commit:
add patch for selinux policy changes

add patch for selinux policy changes to properly run kanto and mosquitto with selinux enabled

Signed-off-by: Francesco Pham <francesco.pham@huawei.com>

Commit:
Update README

- change repo url to eclipse-oniro4openharmony
- update adjusting security settings chapter
